datafusion/physical-plan/src/aggregates/group_values/bytes.rs

```diff
@@ -115,6 +116,11 @@ impl<O: OffsetSizeTrait> GroupValues for GroupValuesByes<O> {
 
                 emit_group_values
             }
+            EmitTo::NextBlock(_) => {
```

Comment:
I think supporting chunked emission in groups would also likely help click bench performance, but is a natural follow on project.